tests/ImageSharp.Tests/Formats/WebP/ColorSpaceTransformUtilsTests.cs

```diff
@@ -71,7 +71,7 @@ private static void RunCollectColorRedTransformsTest()
     public void CollectColorBlueTransforms_WithHardwareIntrinsics_Works() => FeatureTestRunner.RunWithHwIntrinsicsFeature(RunCollectColorBlueTransformsTest, HwIntrinsics.AllowAll);
 
     [Fact]
-    public void CollectColorBlueTransforms_WithoutVector128_Works() => FeatureTestRunner.RunWithHwIntrinsicsFeature(RunCollectColorBlueTransformsTest, HwIntrinsics.DisableSSE41);
+    public void CollectColorBlueTransforms_WithoutVector128_Works() => FeatureTestRunner.RunWithHwIntrinsicsFeature(RunCollectColorBlueTransformsTest, HwIntrinsics.DisableSSE42);
```

Comment:
The entries in this class for `WithoutVector128_Works` tests should be `HwIntrinsics.DisableHWIntrinsic`